test/jdk/java/nio/Buffer/Basic-X.java.template

```diff
@@ -1003,6 +1003,9 @@ public class Basic$Type$
         catchReadOnlyBuffer(b, () -> rb.put(new String(new char[rb.remaining() + 1])));
         catchReadOnlyBuffer(b, () -> rb.append(new String(new char[rb.remaining() + 1])));
 
+        // 8266320
```

Comment:
The bug ID should be deleted here and appended to the list of bug IDs in Basic.java.